agenta-backend/pyproject.toml

```diff
@@ -17,6 +17,7 @@ sqlmodel = "^0.0.8"
 motor = "^3.1.2"
 python-multipart = "^0.0.6"
 openai = "^0.27.8"
+litellm = "^0.1.400"
```

Comment:
No need to add the requirements in the backend. It only needs to be in the requirements.txt files for the example apps